repos/fdbt-site/src/pages/api/capPricingPerDistance.ts

```diff
@@ -0,0 +1,120 @@
+import { startCase } from 'lodash';
+import { NextApiResponse } from 'next';
+import { CAP_PRICING_PER_DISTANCE_ATTRIBUTE } from '../../../src/constants/attributes';
+import { redirectTo } from '../../../src/utils/apiUtils';
+import { isCurrency, isValidNumber } from '../../../src/utils/apiUtils/validator';
+import { updateSessionAttribute } from '../../../src/utils/sessions';
+import { CapDistancePricing, DistanceCap, ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
+
+export const checkInputIsValid = (inputtedValue: string | undefined, inputType: string): string => {
+    let error;
+
+    if (!inputtedValue) {
+        error = `${startCase(inputType)} cannot be empty`;
+    } else if (Math.sign(Number(inputtedValue)) === -1) {
+        error = 'This must be a positive number';
```

Comment:
Empty price error - 'Enter a price for the distance'
Negative number error - 'Prices cannot be negative numbers'